contracts/zero-ex/tests/utils/DeployZeroEx.sol

```diff
@@ -293,7 +323,10 @@ contract DeployZeroEx is Test {
 
         ZERO_EX_DEPLOYED.zeroEx = IZERO_EX;
         isDeployed = true;
-        logDeployedZeroEx();
+        if (ZERO_EX_DEPLOY_CONFIG.logDeployed) {
```

Comment:
why make this optional?

Reply:
That logging is way too noisy imo. If we are using `DeployZeroEx` in `setUp` (as it is probably best used), for every local test we get a large identical dump of addresses. It is unclear to me why we are logging these addresses in the first place, and in the cases I can think of that this would be useful in, some combination of assertions and `vm.label` seems like a better alternative.